ontology/yaml/resources/LIGHTING/entity_types/LT.yaml

```diff
@@ -13,10 +13,26 @@
 # limitations under the License.
 
 
-EXIT:
-  description: "Illuminated exit signs"
-  is_abstract: true
+LT_SS:
+  description: "Standard light."
+  is_canonical: true
   implements:
   - LT
+  - SS
 
+LT_SS_DC:
+  description: "Standard light with dimming control."
+  is_canonical: true
+  implements:
+  - LT
+  - SS
+  - DC 
 
+LT_SS_DC_SC:
+  description: "Standard light with dimming control."
+  is_canonical: true
+  implements:
+  - LT
+  - SS
+  - DC 
```

Comment:
The lights i've seen with dimming don't have SS - oof is just dimmer = 0%.  (enlighted).  Is this based on an actual example?

Reply:
not specifically. i would imagine we will run into it eventually. theres nothing technically incorrect with this representation, so if we end up with lights with only dimmer % we would just make a simpler type.

Reply:
Conclusion: it is valid for lights to be "on" with dimmer of 0%